### ISSUE TITLE
Add support for sub-expressions 

### ISSUE THREAD

**Issue by cryarchy** [mohd-akram/vscode-html-format#12]
This is the best vscode Handlebars formatted that I have found (and I have tried many). Thank you.

Currently, it will reformat this:

```hbs
<li>
  <a {{#if (equals pageLink "/users" ) }} class="is-active" {{/if}} href="/users">
     Users
  </a>
</li>
```

to:

```hbs
<li>
  <a {{#if (equals pageLink"/users" ) }} class="is-active" {{/if}} href="/users">
     Users
  </a>
</li>
```

removing the space between the `pageLink` variable and the `"/user"` literal.

**Comment by cryarchy** [comment]
The issue seems to happen with double-quoted string literals only. The following snippets work:

```hbs
<li>
  <a {{#if (equals pageLink 42 ) }} class="is-active" {{/if}} href="/users">
     Users
  </a>
</li>
```

and:

```hbs
<li>
  <a {{#if (equals pageLink anotherVariable ) }} class="is-active" {{/if}} href="/users">
     Users
  </a>
</li>
```

and surprisingly this too:

```hbs
<li>
  <a {{#if (equals pageLink 'single-quote' ) }} class="is-active" {{/if}} href="/users">
     Users
  </a>
</li>
```

**Comment by mohd-akram** [comment]
Sorry about that. This is fixed in v0.0.5.
